vipps-checkout-api.md

```diff
@@ -1,52 +1,121 @@
-# Vipps Checkout guide
+# Vipps Checkout
 
-Vipps Checkout is designed to be a low friction, low complexity flow where Vipps Checkout ensures a smooth and efficient checkout experience using the trusted Vipps technology and brand.
-
-Preliminary documentation. Subject to change
+Vipps Checkout provides an all-in-one solution for receiving payment for goods and services online using trusted Vipps technology and brand. It combines other Vipps products, Vipps Login and Vipps eCom/ePayments, allowing a frictionless integration for merchants.
 
 API version: 2.0.0.
 
-Document version: 1.0.1.
```

Comment:
I think we need this 'document verision', too

Reply:
Not sure how to version the document. Do you consider this a major or minor update?

Reply:
I think minor is good (1.1.0) :-)

Reply:
1.0.2 is probably also fine, if their code does not need to be changed.